lib/rules/web/console/base/login.xyaml

```diff
@@ -43,36 +42,36 @@ login_rhsso:
         id: username
       op: send_keys <username>
       type: input
+    - selector:
+        id: login-show-step2
+      op: click
+      type: button
     - selector:
         id: password
       op: send_keys <password>
       type: input
-  elements:
     - selector:
-        value: Log in
-      op: click
+        id: kc-login
       type: input
-
+      op: click
 login_console:
   elements:
     - selector:
-        id: inputUsername
+        id: username
       op: send_keys <username>
       type: input
     - selector:
-        id: inputPassword
+        id: login-show-step2
+      op: click
+      type: button
+    - selector:
+        id: password
       op: send_keys <password>
       type: input
-  scripts:
-  - command: return document.getElementsByName("csrf").length > 0
-    expect_result: true
-  cookies:
-  - name: csrf
-    expect_result: true
   elements:
     - selector:
-        text: Log In
-      type: button
+        id: kc-login
```

Comment:
it looks like the issue with `login_page_loaded` is with this change here. I think you can add this id as `#kc-login` to the list under `login_page_loaded` rule and you would get things working without removing the check.